src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/AbstractCreateSDORespondentNotificationSender.java

```diff
@@ -43,6 +41,15 @@ void notifyRespondentPartySDOTriggered(CaseData caseData) {
         }
     }
 
+    private String getSDOTemplate(CaseData caseData) {
```

Comment:
As discussed, it would be better to only handle bilingual for spec claims so better to have a nested if